app/services/service_health/monitor.py

```
@@ -0,0 +1,220 @@
+                open_circuits.append(rule)
+        return {
+            "is_paused": len(open_circuits) > 0,
+            "open_circuits": open_circuits,
+        }
+
+    async def run_auto_health_check(self) -> None:
+        """Evaluate all circuits and auto-reset if clean."""
+        if not await ENABLE_SERVICE_HEALTH_MONITORING():
+            return
+
+        for rule, circuit in _CIRCUITS.items():
```

Ruff (0.15.12)

[warning] Loop control variable `rule` not used within loop body

Rename unused `rule` to `_rule`

(B007)